hyperspy/signal.py

```diff
@@ -1566,8 +1566,9 @@ def _export_factors(self,
                 if folder is not None:
                     filename = os.path.join(folder, filename)
                 ensure_directory(filename)
-                fac_plots[idx].savefig(filename, save_figures_format,
-                                       dpi=600)
+                _args = {'dpi': 600,
+                         'format': save_figures_format}
+                fac_plots[idx].savefig(filename, **_args)
```

Comment:
At first I though `save_figures_format` was a dictionary itself and needed updating, but apparently it's just a string. I left it like so if in the future we want to enable passing user arguments. Then `_args` can just be updated with the user-supplied dict.

Too far-fetched?